include/dsn/cpp/auto_codes.h

```diff
@@ -167,56 +167,6 @@ namespace dsn
 #endif
     };
 
-    class atom_int
```

Comment:
Maybe we need a conversation to talk about all these things: the message parser, the meta server, and some details in the serialization.